cloud/parameterstore/parameter_record_db.go

```diff
@@ -0,0 +1,76 @@
+package parameterstore
+
+import (
+	"context"
+	"time"
+
+	"github.com/mongodb/anser/bsonutil"
+	adb "github.com/mongodb/anser/db"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Collection holds metadata information about parameters.
+const Collection = "parameter_records"
+
+var (
+	nameKey        = bsonutil.MustHaveTag(ParameterRecord{}, "Name")
+	lastUpdatedKey = bsonutil.MustHaveTag(ParameterRecord{}, "LastUpdated")
+)
+
+// BumpParameterRecord bumps the parameter record to indicate that the parameter
+// was changed. information. This will not modify if the record exists and its
+// most recent update is more recent than lastUpdated.
+func BumpParameterRecord(ctx context.Context, db *mongo.Database, name string, lastUpdated time.Time) error {
+	res, err := db.Collection(Collection).UpdateOne(ctx, bson.M{
```

Comment:
There's no need for an index for this particular query because finding by name is finding by the doc's automatically-indexed `_id`, which matches a single document anyways.